### DISCUSSION TITLE
Can I show the results of the report as badge in my Github Repo?

### DISCUSSION THREAD

**Discussion by klacol** [DannyDainton/newman-reporter-htmlextra#388] [Q&A] [answered]
Hi, we use a lot the badges of shields.io in our repos. A badge offers three things:

1. The user knows, that there is some external services attached to this repo (e.g. a build)
2. The user can click on the badge a go to this service
3. The user can see the result of the external service, that is indicated by the color of the badge (green/red)

Is there the possibility to create an badge of a postman/newman API monitor?

**Comment by DannyDainton** [comment, marked as answer]
That's outside of the scope of this or any other community reporter to be honest. 

Might be worth raising as a feature request on the Newman repo. 

If Newman is running the Collection in a CI context, you could get a build badge that way but it wouldn't associated with the htmlextra report.

**Reply by klacol** [reply]
Thanks
